frontend/control-center/src/pages/Connectors/ConnectedChannelsList/index.tsx

```diff
@@ -198,6 +198,7 @@ const ConnectedChannelsList = () => {
       </div>
       <Pagination
         totalCount={filteredChannels.length}
+        pageSize={8}
```

Comment:
Why remove it from the component to pass it as a prop?

Reply:
because we have different pageSizes for different lists (e.g. ContactsList (9), ChannelsList (8)